ProjFS.Mac/PrjFSKext/public/Message.h

```diff
@@ -8,13 +8,9 @@
 typedef enum
 {
     MessageType_Invalid = 0,
-    
-    // Messages from user mode to kernel
-    MessageType_UtoK_StartVirtualizationInstance,
-    MessageType_UtoK_StopVirtualizationInstance,
-    
+	
     // Messages from kernel to user mode
-    MessageType_KtoU_EnumerateDirectory,
+    MessageType_KtoU_EnumerateDirectory = 3,
```

Comment:
Is there any reason why we need to set this to 3?

Reply:
We don't have a proper API versioning story yet (there's a version check, but we're not incrementing the version), so not renumbering messages seemed a lower risk strategy, although I suppose as we're no longer guaranteeing paths on some messages that will only ensure backwards compatibility one way (new userlib with older kext) not both ways.

Reply:
Rather than renumbering could you manually bump `PrjFSKextVersion`?

We have #648 open to starting doing this automatically.

Reply:
Sure, can do.